srcpkgs/m4rie/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'm4rie'
+pkgname=m4rie
+version=20200125
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config autoconf automake libtool"
+makedepends="m4ri"
```

Comment:
You will want to makedepend on `m4ri-devel` once you create the subpkg as mentioned before.